models/forms/class.XmlEditForm.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+
+declare(strict_types=1);
+
+class taoQtiTest_models_forms_XmlEditForm extends tao_helpers_form_FormContainer
+{
+    public function __construct(core_kernel_classes_Resource $test, $xmlString, $options = [])
+    {
+        parent::__construct(
+            [
+                'id' => $test->getUri(),
+                'xmlString' => $xmlString
+            ],
+            $options
+        );
+    }
+
+    /**
+     * @return mixed|void
+     * @throws common_Exception
+     * @throws Exception
+     */
+    public function initForm()
```

Comment:
same as above
plus is there any need to have less strict access to this method?
UPD:
plus you can use the `{@inheritdoc}` as the docblock comment